### PR TITLE
Remove Alibaba Cloud support

### PR DESCRIPTION
**Pull request by 2uasimojo**
[HIVE-2412](https://issues.redhat.com//browse/HIVE-2412)

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/hold

I don't want to have to backport this, so let's wait until we're reasonably clear of the fixes and features we know we're going to want in mce-* branches soonish.

**Comment by 2uasimojo** [comment]
/assign @suhanime 

(Mark on vacation)

File: pkg/controller/clusterpool/clusterpool_controller_test.go
Comment (suhanime) [inline pkg/controller/clusterpool/clusterpool_controller_test.go:1]: The changes in this file don't seem Alibaba specific? It's okay though, I see it's just an improvement.

**Review by suhanime — COMMENTED** [review]
/lgtm

**Pull request merged**